fl4health/model_bases/pca.py

```diff
@@ -35,32 +63,40 @@ def forward(self, X: Tensor, center_data: bool) -> Tuple[Tensor, Tensor]:
 
         Args:
             X (Tensor): Data matrix.
-            center_data (bool): If true, then the data mean will be subtracted from all data points prior to
-            performing PCA.
+            center_data (bool): If true, then the data mean will be subtracted
+            from all data points prior to performing PCA. If center_data is false,
+            it is expected that the data has already been centered and an exception
+            will be thrown if it is not.
 
         Returns:
-            Tuple[Tensor, Tensor]: The principal components (i.e., left singular vectors) and their corresponding
-            singular values.
+            Tuple[Tensor, Tensor]: The principal components (i.e., right singular vectors)
+            and their corresponding singular values.
+
+        Note: the algorithm assumes that the rows of X_prime are the data points.
```

Comment:
Just a small thing. `X_prime` is only mentioned in the code, but in generated documentation, the user will only be aware of X. So unless we introduce X_prime briefly, it might be confusing.

Reply:
Good point. I think it might be clearer to just use `X` in the documentation then (mentioning that we assume it is reshaped as needed)